lib-ee/emqx_ee_bridge/src/kafka/emqx_bridge_impl_kafka_producer.erl

```diff
@@ -185,9 +215,14 @@ render_timestamp(Template, Message) ->
             erlang:system_time(millisecond)
     end.
 
-on_kafka_ack(_Partition, _Offset, _Extra) ->
-    %% Do nothing so far.
-    %% Maybe need to bump some counters?
+%% Wolff producer never gives up retrying
+%% so there can only be 'ok' results.
+on_kafka_ack(_Partition, Offset, {ReplyFn, Args}) when is_integer(Offset) ->
+    %% the ReplyFn is emqx_resource_worker:reply_after_query/8
+    apply(ReplyFn, Args ++ [ok]);
```

Comment:
Small nit, but just remembered that we have `emqx_resource:apply_reply_fun(ReplyFunAndArgs, Result)`.